releasenotes/notes/statevector-equiv-ad588688fa260940.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Allow constructing :class:`~qiskit.quantum_info.Statevector` objects from circuits, assuming
+    that the qubits are initialized in :math:`|0\rangle`. This also allows to call
+    ``Statevector.equiv`` with a circuit as argument.
```

Comment:
I don't think its necessary to mention the `equiv` part here. Just:
>Allow constructing :class:`~qiskit.quantum_info.Statevector` objects from circuits, assuming
    that the qubits are initialized in :math:`|0\rangle`.

Reply:
done in 5c59880